### ISSUE TITLE
[ED-med] "Language" and "Country" filters - Why clickable text below?

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-course-list#281]
It's not clear why the summary of languages and countries is clickable text (`tabindex="0"`). Seems unnecessary; if you want it to be read by screen readers while people are tabbing through the fields, make it an `aria-describedby` attribute. 

Also, the placement below the `select` fields seems not as useful as above them. This information is most useful before I expand the menu list so I'll know what to expect and then there's not much need to make this dynamic because you put the numbers in the answer choices (as in "English (2)"). So overall the information (if needed) seems to have an unclear reading order (though I get that you put it below because the current implementation updates after you make a selection).

**Comment by leticiaseixas** [comment]
- This information is updated after a filter selection. It just seems odd to me to have this information before the field, especially as the other fields have this total information being displayed after the field text itself.
- Also, the aria-describedby, in this case, does not work, since it is a dynamic information. That is, when this information is updated, screen reader users would not be aware of this change, as the field, including the aria-describedby, has already been read - as you mentioned.
- Finally, we think that this additional information is useful because the total displayed for each item in each select list can be confusing, since each course can have several languages and countries. In the case of fields with a checkbox, this visualization is easier. For this select list it was confusing for some users to have, for instance, Portugal (1), France (1), and only one result appearing in the list. With that, we aim to make it more clear that we have "1 course available from providers in 2 different countries".

**Comment by MicheleAWilliams-A11y** [comment]
Thanks for the explanations, these help. I'd say consider this closed if there's no other objections.
